backend/lib/endpoints/posts.js

```diff
@@ -341,10 +342,17 @@ async function routes(app) {
           numComments = commentQuery[0].comments;
         }
       }
+
+      const projectedPost = {
+        ...post.toObject(),
+        liked: post.likes.includes(mongoose.Types.ObjectId(userId)),
+        likesCount: post.likes.length,
+      };
+
       return {
         comments,
         numComments,
-        post,
+        projectedPost,
```

Comment:
```suggestion
        post: projectedPost,
```

Renaming the return element will break the frontend. I think the above fix will work.

Reply:
Haha I thought about this, but wasn't sure.

Reply:
I'm going to make the commit here and test on review branch. Everything else is good.